api/models/SdtdConfig.js

```diff
@@ -178,6 +178,21 @@ module.exports = {
       defaultsTo: false
     },
 
+    enabledVote: {
+      type: 'boolean',
+      defaultsTo: false
+    },
+
+    voteForServerServerKey: {
```

Comment:
Maybe we should rename this to something clearer? Like serverListAPIKey(s) or something?

Reply:
The field's main purpose is for one key, maybe just add S at the end?